scripts/fusion_pipeline/2_1_fusion.py

```diff
@@ -0,0 +1,185 @@
+"""Fusion Part 2.1: Scale the image to the required scales."""
+
+##################################################################################################
+#                                           IMPORTS
+##################################################################################################
+import argparse
+import time
+from typing import Any
+
+import dask
+import dask.array as da
+import numpy as np
+import skimage.transform
+import zarr
+
+parser = argparse.ArgumentParser()
+
+# External function copied from: https://github.com/ome/ome-zarr-py/blob/5c5b45e46e468a3f582a583c915b9eefb636b82c/ome_zarr/dask_utils.py#L11C1-L62C63
+
+
+def resize(
+    image: da.Array, output_shape: tuple[int, ...], *args: Any, **kwargs: Any
+) -> da.Array:
+    r"""
+    Resize function.
+
+    Wrapped copy of "skimage.transform.resize"
+    Resize image to match a certain size.
+
+    :type image: :class:`dask.array`
+    :param image: The dask array to resize
+    :type output_shape: tuple
+    :param output_shape: The shape of the resize array
+    :type \*args: list
+    :param \*args: Arguments of skimage.transform.resize
+    :type \*\*kwargs: dict
+    :param \*\*kwargs: Keyword arguments of skimage.transform.resize
+    :return: Resized image.
```

Comment:
Please use a numpy-style docstring like you did above in `radius_map_generator_gpu()`